src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -620,6 +620,13 @@ public void CopyTo(byte* ptr)
             Memory.Copy(ptr, _mem, _size);
         }
 
+        public BlittableJsonReaderObject Clone(JsonOperationContext context)
```

Comment:
I have to clone the conflicted document (which is selected as the resolver), before I pass it to the Put method.
Since, all conflict will be deleted before the resolved document will be saved in the storage.
Otherwise, this document will be also deleted.